src/lsm/forest.zig

```diff
@@ -438,7 +438,6 @@ pub fn ForestType(comptime _Storage: type, comptime groove_cfg: anytype) type {
             const forest = @fieldParentPtr(Forest, "manifest_log", manifest_log);
             assert(forest.progress.? == .checkpoint);
             assert(forest.manifest_log_progress == .idle);
-            forest.grid.assert_only_repairing();
```

Comment:
There may (now) be non-repair grid-writes while we are flushing the manifest log blocks – we are concurrently flushing the trailers at the same time.